Scripts/travis-build-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -x -o pipefail
+
+if [[ "$SWIFT_BUILD" == "true" ]]; then
+    swift build
```

Comment:
This is only running `build` and not `test` because of what you mentioned in this comment, correct? https://github.com/venmo/DVR/pull/73#issuecomment-396842789

Reply:
yes exactly 👌